docs/element-definitions.md

```diff
@@ -252,73 +281,102 @@ transitions.
 
 - `element`: category (string, fixed)
 - `meta`
-    - `classes` (array, fixed)
-        - (enum[string])
-            - api - Category is a API top-level group.
-            - resourceGroup - Category is a set of resource.
-            - dataStructures - Category is a set of data structures.
-            - scenario - Category is set of steps.
-            - transitions - Category is a group of transitions.
-            - authSchemes - Category is a group of authentication and authorization scheme definitions
+    - `classes` (Array Element)
+        - `content` (array, fixed-type)
+            - One Of
```

Comment:
I am not sure if the syntax here is correct I understand what is being said but just feels strange. The `One Of` String Elements is just strange.